R-package/pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,93 @@
+url: https://github.com/dmlc/xgboost
+
+template:
+  bootstrap: 5
+  math-rendering: mathjax
```

Comment:
@trivialfis What about the `katex` with the workaround that is mentioned in the issue you linked?
https://github.com/r-lib/pkgdown/issues/2704#issuecomment-2307055568

Looks like it should solve the issue of symbols like sqrt showing random glitches.

Reply:
Will test it.

Reply:
Let's not do that. Injecting npm dependencies with a fixed version is outside my comfort zone. I know nothing about web development and would like to avoid it in XGBoost.